File: Free Html To Pdf Converter for ASP.NET MVC/[C#]-Free Html To Pdf Converter for ASP.NET MVC/C#/SelectPdf.Samples/Controllers/ViewHttpPostDataController.cs
using System;
using System.Text;
using System.Collections.Specialized;
using System.Web.Mvc;

namespace SelectPdf.Samples.Controllers
{
    public class ViewHttpPostDataController : Controller
    {
        // GET: ViewHttpPostData
        public ActionResult Index()
        {
            StringBuilder output = new StringBuilder();
            output.Append("<br/>Request method: " +
                Request.HttpMethod + "<br/>");

            // Load POST form fields collection.
            NameValueCollection form = Request.Form;

            // Put the names of all keys into a string array.
            String[] keys = form.AllKeys;
            for (int i = 0; i < keys.Length; i++)
            {
                output.Append("Name: " + keys[i] + "<br/>");

                // Get all values under this key.
                String[] values = form.GetValues(keys[i]);
                for (int j = 0; j < values.Length; j++)
                {
                    output.Append("Value: " + values[j] + "<br/>");
                }
                output.Append("<br/>");
            }

            ViewData.Add("PostDataValue", output.ToString());

            return View();
        }
    }
}

File: Free Html To Pdf Converter for ASP.NET MVC/[VB]-Free Html To Pdf Converter for ASP.NET MVC/VB/SelectPdf.Samples/Controllers/ViewHttpPostDataController.vb
Namespace Controllers
    Public Class ViewHttpPostDataController
        Inherits Controller

        ' GET: ViewHttpPostData
        Function Index() As ActionResult
            Dim output As New StringBuilder()
            output.Append("<br/>Request method: " +
                          Request.HttpMethod + "<br/>")

            ' Load POST form fields collection.
            Dim form As NameValueCollection = Request.Form

            ' Put the names of all keys into a string array.
            Dim keys As [String]() = form.AllKeys
            For i As Integer = 0 To keys.Length - 1
                output.Append("Name: " + keys(i) + "<br/>")

                ' Get all values under this key.
                Dim values As [String]() = form.GetValues(keys(i))
                For j As Integer = 0 To values.Length - 1
                    output.Append("Value: " + values(j) + "<br/>")
                Next
                output.Append("<br/>")
            Next

            ViewData.Add("PostDataValue", output.ToString())

            Return View()
        End Function
    End Class
End Namespace